### ISSUE TITLE
<root>/msg is missing

### ISSUE THREAD

**Issue by GiulliMo** [moeelm/voice_ros#1]
Hi

while doing catkin_make it says Voice-Control-ROS/msg not found. 

Giulli

**Comment by Femi-Tofade** [comment]
How did you solve this?

**Comment by GiulliMo** [comment]
I took an other node 😂👌

**Comment by Femi-Tofade** [comment]
I dont understand..
please can you explain better?
and were you able to do the voice navigation of the robot?

Please help me, i am tired and mentally tired at the moment

**Comment by GiulliMo** [comment]
Yes we use MATLAB/Simulink with deep learning Network Toolbox. Connecting the result of the given example (Matlab example) with our moving and navigation system.

**Comment by Femi-Tofade** [comment]
This works with the ros navigation system?
and able to move the robot from one destination to another using voice?
Can you please share? 
WHat is your Linkedin or email please?
This is the last stage of my project, please help me

**Comment by GiulliMo** [comment]
I have documented only my navigation stack in german but probably you can understand it by looking at the pictures 👌😂

**Comment by Femi-Tofade** [comment]
haha, my  email is tolatof@gmail.com
Thanks

I need to integrate voice control with the navigation stack,,

**Comment by GiulliMo** [comment]
Mail incoming

**Comment by Femi-Tofade** [comment]
Thanks a lot

**Comment by yani-rl-ai** [comment]
hi @GiulliMo can i get Voice-Control-ROS/msg ? my email : yani.muhammad97@gmail.com

**Comment by sm5278** [comment]
Hi, @GiulliMo 
can i get Voice-Control-ROS/msg ?  my email is zxcvbnm61201@gmail.com
Thanks!
